recipe/meta.yaml

```
@@ -39,21 +38,18 @@ requirements:
     - python                                # [build_platform != target_platform]
     - pip                                   # [build_platform != target_platform]
     - cross-python_{{ target_platform }}    # [build_platform != target_platform]
-    - numpy 1.26.*                          # [build_platform != target_platform]
-    - qt6-main >=6.6.1,<7.0.0               # [build_platform != target_platform]
+    - qt6-main                              # [build_platform != target_platform]
   host:
     - python
-    - numpy 1.26.*
-    - qt6-main >=6.6.1,<7.0.0
+    - qt6-main
     - packaging
     - pip
     - setuptools
     - wheel
     - xorg-libxfixes  # [linux]
   run:
     - python
-    - qt6-main >=6.6.1,<7.0.0
-    - numpy >=1.26,<2.0
+    - qt6-main
```

Do we still need NumPy at runtime? Asking because of [this NumPy requirement]( https://github.com/nion-software/nionui-tool/blob/5aeb713d13742e76907ab9c3426d336487279b3e/nionui_app_test/setup.cfg#L6-L7 )

```suggestion
    - qt6-main
    - numpy >2.0,<3
```

---

Numpy is not needed anymore. Also: Sorry I force pushed a new version which restores the patch to only build the non-fat versions. Building now.